src/storage/test/StorageClientTest.cpp

```diff
@@ -42,14 +42,24 @@ TEST(StorageClientTest, VerticesInterfacesTest) {
     auto addrsRet
         = network::NetworkUtils::toHosts(folly::stringPrintf("127.0.0.1:%d", localMetaPort));
     CHECK(addrsRet.ok()) << addrsRet.status();
+    auto& addrs = addrsRet.value();
+    uint32_t localDataPort = network::NetworkUtils::getAvailablePort();
+    auto hostRet = nebula::network::NetworkUtils::toHostAddr("127.0.0.1", localDataPort);
+    auto& localHost = hostRet.value();
     auto mClient
-        = std::make_unique<meta::MetaClient>(threadPool, std::move(addrsRet.value()), true);
-    mClient->init();
+        = std::make_unique<meta::MetaClient>(threadPool, std::move(addrs), localHost, true);
+    LOG(INFO) << "Add hosts and create space....";
+    auto r = mClient->addHosts({HostAddr(localIp, localDataPort)}).get();
+    ASSERT_TRUE(r.ok());
+    mClient->waitForMetadReady();
+    while (meta::ActiveHostsMan::instance()->getActiveHosts().size() == 0) {
```

Comment:
Do we need this? If MetaClient has send heartbeat successfully, it should have been an active host.